.env.development

```diff
@@ -0,0 +1 @@
+REACT_APP_BACKEND_URL = 'http://private-anon-b00bb0b7e8-wbooksapi.apiary-mock.com/api/v1'
```

Comment:
this is ok in this training, but in a real project this should never be pushed to the repo, and. in the `.gitignore` should be a rule like `.env.*` to avoid pushing this into the repo